BucStop/Controllers/SnakeController.cs

```diff
@@ -0,0 +1,12 @@
+﻿using Microsoft.AspNetCore.Mvc;
+
+namespace BucStop.Controllers
+{
+    public class SnakeController : Controller
+    {
+        public IActionResult Index()
+        {
+            return View();
```

Comment:
Do we need a whole controller for snake? We could just make a controller for games, and add a IActionResult method for Snake to take you to the snake page. The URL in this example would be 1.1.1.1/games/snake